swagger.json

```diff
@@ -5101,15 +5101,41 @@
                 "label",
                 "value"
             ],
+            "description": "A field of a document. A field can be a string type, number type, or photo type. Between stringValue, numberValue, and photoValue, only one can exist for a single document field depending on the field's type.",
             "properties": {
                 "label": {
                     "type": "string",
                     "description": "Descriptive name of this field.",
                     "example": "Fuel Cost ($)"
                 },
                 "value": {
-                    "description":  "Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
+                    "description":  "DEPRECATED: Please use stringValue, numberValue, or photoValue instead. Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
                     "example": 23.45
+                },
+                "stringValue": {
+                  "type": "string",
+                  "description": "Value of this field if this document field is a string type field.",
+                   "example": "This is a string."
+                },
+                "numberValue": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Value of this field if this document field is a number type field.",
+                   "example": 12.34
+                },
+                "photoValue": {
+                  "type": "array",
+                 "description": "Value of this field if this document field is a photo type field. Array of photo objects where each object contains a URL for a photo.",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "url": {
+                        "type": "string",
+                        "description": "Photo URL"
```

Comment:
What type of photo is this expected to return? (JPG, PNG, etc).

Reply:
Does a photo not have a name or description associated with it?

Reply:
I think adding the type of the returned photo (jpg) to the description would be worthwhile. Are there any downsides to exposing the UUID?

Reply:
I hope not, since I assume the URL will include the `uuid` in the path. Is there also a human assigned name / description to each photo? In other words, can the driver annotate each photo?

Reply:
I don't think there are any downsides. There is not a human assigned name for the photo.